docs/src/pages/components/menu.mdx

```diff
@@ -47,12 +62,16 @@ The `Trigger`has also a prop for `disabled`
 
 ### Simple Menu
 
+In this example you can see a simple `<Menu>` with some items to select.
+
 ```tsx preview file=components/Menu/menu-selected.demo.tsx
 
 ```
 
 ### Disabled Menu
 
+In this example the `<Menu.Trigger>` has set its prop `disabled`. So you can't interact with the `<Menu>` anymore.
```

Comment:
If the Menu is disabled then we should change the cursor to not-allowed? `cursor: not-allowed;`

Reply:
hm i don't know. I also have the question in my mind: why would i disable a complete menu? :D